zerver/lib/emoji.py

```diff
@@ -14,6 +14,31 @@
 NAME_TO_CODEPOINT_PATH = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "name_to_codepoint.json")
 CODEPOINT_TO_NAME_PATH = os.path.join(settings.STATIC_ROOT, "generated", "emoji", "codepoint_to_name.json")
 
+# Emoticons and which emoji they should become. Duplicate emoji are allowed.
+# Changes here should be mimicked in `static/js/emoji.js`
+# and `templates/zerver/help/enable-emoticon-translations.md`.
+EMOTICON_CONVERSIONS = {
+    ':)': ':smiley:',
+    '(:': ':smiley:',
+    ':(': ':slightly_frowning_face:',
+    '<3': ':heart:',
+    ':|': ':expressionless:',
+    ':/': ':confused:',
+}
+
+possible_emoticons = EMOTICON_CONVERSIONS.keys()
```

Comment:
Do we need this variable?

Reply:
@timabbott Yes, because it is used to create `emoticon_regex` which is in turn [used in the Markdown pattern](https://github.com/zulip/zulip/pull/8081/files/7b51a3a130f668e97816e6a008f9f453d76b4128#diff-237f2d7c30a5fdd38c7f1e2d8f6734eeR1594).